testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/GrettySmokeTest.groovy

```diff
@@ -32,6 +33,8 @@ import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
 @Requires(UnitTestPreconditions.Jdk11OrLater)
 class GrettySmokeTest extends AbstractPluginValidatingSmokeTest {
 
+    // There is no version of Gretty compatible with 9.0
+    @Ignore("https://github.com/gretty-gradle-plugin/gretty/issues/312")
```

Comment:
💭 I don't think that Gretty plugin is something we should block removal of the method on.

Reply:
Agreed